transforms/map-filter/mapFilterTransform.js

```
@@ -9275,60 +9298,868 @@ function v4(options, buf, offset) {
+    }
+    return Math.abs(hash);
+  }
+  validateKey(key) {
+    if (!key || typeof key !== "string") {
+      return false;
+    }
+    if (key.length > 255) {
+      console.warn(`Key is quite long (${key.length} chars): ${key.substring(0, 50)}...`);
+      return false;
+    }
+    const problematicChars = /[<>:"/\\|?*\x00-\x1f]/;
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)